Makefile

```
@@ -269,11 +271,14 @@ all-containerized:
 	docker rm $(BUILD_CONTAINER_NAME)
 	docker rmi $(BUILD_CONTAINER_IMAGE):$(VERSION)
 
-# Make both linux and windows binaries
+# Make all platform binaries
 .PHONY: all-binaries-platforms
 all-binaries-platforms:
-	export GOOS=linux; make all-binaries
-	export GOOS=windows; make all-binaries
+	for goos in $$(echo $(GOOS) | sed "s/,/ /g"); do \
+		for goarch in $$(echo $(GOARCH) | sed "s/,/ /g"); do \
+			make all-binaries GOOS=$$goos GOARCH=$$goarch; \
+		done \
+	done
```

if the env vars are already arrays, we can simplify the array iteration significantly:
```suggestion
	@for goos in "$(GOOSES)"; do \
		for goarch in "$(GOARCHES)"; do \
			make all-binaries GOOS=$$goos GOARCH=$$goarch; \
		done \
	done
```

---

we can get this done by setting these default vars in the makefile: 
```
GOOSES ?= "linux windows"
GOARCHES ?= "amd64 arm64"
```

that makes the default invocation:
`make all-binaries-multiplatform`.

to override it you need to set those as _Make_ variables (not shell variables), to different strings: 
`make all-binaries-multiplatform GOOSES="\"solaris bsd\"" ...`

---

worth leaving a comment that those strings need to be escaped to override correctly

---

Thanks, attempted in the latest commit, let me know if that's what you mean or if I should change something